flow360/component/simulation/translator/solver_translator.py

```diff
@@ -705,6 +733,97 @@ def get_solver_json(
         )
 
     ##:: Step 10: Get heat transfer zones
+    solid_zone_boundaries = set()
+    if has_instance_in_list(input_params.models, Solid):
+        translated["heatEquationSolver"] = {
```

Comment:
We need validation that there is only 1 Solid model.

Reply:
Why? The unit test has 3

Reply:
Then the `get_attribute_from_instance_list` will only get the setting from the first solid? How the 2nd and 3rd setting gets translated into solver JSON?

Reply:
Validation will fail if these settings are different